lib/sycamore/sycamore/transforms/table_structure/table_transformers.py

```diff
@@ -202,11 +205,21 @@ def nms_by_containment(container_objects, package_objects, overlap_threshold=0.5
 
 
 def slot_into_containers(
-    container_objects, package_objects, overlap_threshold=0.5, unique_assignment=True, forced_assignment=False
+    container_objects,
+    package_objects,
+    overlap_threshold=0.5,
+    unique_assignment=True,
+    forced_assignment=False,
+    _early_exit_vertical=False,  # yes. see docstring.
```

Comment:
why yes?

Reply:
bc when you read this you go "why?" to which the correct answer is "yes"